app/models/spree/shipment.rb

```diff
@@ -141,15 +142,6 @@ def currency
       order ? order.currency : Spree::Config[:currency]
     end
 
-    # The adjustment amount associated with this shipment (if any)
-    #   Returns only the first adjustment to match the shipment
-    #   There should never really be more than one.
-    def cost
-      adjustment ? adjustment.amount : 0
-    end
-
-    alias_method :amount, :cost
```

Comment:
How was this working previously? We had both an alias and a method defined for `cost`?

Reply:
Yeah there's a weird thing up to Spree 2.1 where shipments have a `cost` field in the database, but it's always `nil` (it never gets set anywhere in the code). The `cost` field will now be used, and the shipment's cost (the shipping fee) will be persisted when it's updated, so a shipment will know it's own cost without having to ask the adjustment.